README.md

```diff
@@ -84,9 +84,9 @@ LiteFarm is comprised of three applications which all reside in this monorepo.
 
 The applications are configured with environment variables stored in `.env` files. Configuration information includes secrets like API keys, so the `.env` files are not included in this git repository.
 
- This repository only contains `.env.default` files for api and webapp. To join the LiteFarm team and recieve full versions of the environment files contact community@litefarm.org. 
-
- Once you recieve the `.env` files, you will have to rename them correctly and place them in the right folders.
+ This repository contains a `.env.default` file for api and webapp. These files contain directions to acquire the personal keys needed to get LiteFarm running locally. Please note you will want to rename these files to `.env` once you have the keys in place. 
```

Comment:
Small detail but I think this will actually happen -- `.env.default` is git tracked so you can't just rename them without git thinking they are deleted. They should be copied and the copy renamed to `.env`

Reply:
Ah yes, good point